src/themes/resetcss/globals/reset.overrides

```diff
@@ -91,8 +91,8 @@ video {
     margin: 0;
     padding: 0;
     border: 0;
-    font-size: 100%;
     font: inherit;
+    font-size: 100%;
```

Comment:
this fixes `declaration-block-no-shorthand-property-overrides` stylelint rule

is this change ok or does it have any side effects?

Reply:
i think, this change is ok.